examples/autoencoder.py

```diff
@@ -42,10 +44,31 @@ def forward(self, x, edge_index):
         elif args.model in ['VGAE']:
             return self.conv_mu(x, edge_index), self.conv_logstd(x, edge_index)
 
+          
+# Linear encoder from linear graph AE and VAE models: https://arxiv.org/abs/1910.00942
```

Comment:
At first, I considered putting all models in the existing (only) `Encoder` class, but quickly realized that I would end up with several if conditions (e.g. as the layer sizes differ), and that creating two classes improves the overall readability.